wdl/GetTruthOverlap.wdl

```diff
@@ -0,0 +1,232 @@
+        input:
+            test_vcfs=test_vcfs,
+            truth_vcfs=truth_vcfs,
+            vapor_files=vapor_files,
+            vapor_size_gib=select_first([vapor_size_gib, 0.0]),
+            ped_files=ped_files,
+            optimal_overlap_cutoffs=optimal_overlap_cutoffs,
+            get_truth_overlap_args=get_truth_overlap_args,
+            sv_utils_docker=sv_utils_docker,
+            max_test_records=max_test_records,
+            max_truth_records=max_test_records
+    }
+
+    output {
+        File truth_overlap_info = GetTruthOverlapTask.truth_overlap_info
+        File output_optimal_overlap_cutoffs = GetTruthOverlapTask.output_optimal_overlap_cutoffs
+    }
+}
+
+
+task MaxInts {
+    input {
+        Array[Int] ints
+    }
+
+    command <<<
+        awk 'BEGIN {z=0;} {z=(z>=$0?z:$0);} END {print z;}' "~{write_lines(ints)}"
+    >>>
+
+    output {
+        Int max_int = read_int(stdout())
+    }
+
+    runtime {
+        docker: "ubuntu:latest"
+        cpu: 1
+        preemptible: 3
+        max_retries: 0
+        memory: "1 GiB"
+        disks: "local-disk 10 HDD"
+    }
+}
+
+
+task GetVcfSize {
+    input {
+        File vcf
+        File? vcf_index
+        String sv_base_docker
+    }
+
+    parameter_meta {
+        vcf: {
+          localization_optional: true
+        }
+    }
+
+    Int disk_gb = round(10 + size(vcf_index, "GiB"))
+    String num_records_file = "num_records.txt"
+    String num_samples_file = "num_samples.txt"
+    # if vcf_index is not supplied, try this path automatically:
+    String automatic_vcf_index = vcf + ".tbi"
+
+    runtime {
+        docker: sv_base_docker
+        cpu: 1
+        preemptible: 3
+        max_retries: 0
+        memory: "2 GiB"
+        disks: "local-disk " + disk_gb + " HDD"
+    }
+
+    command <<<
+        set -eu -o pipefail
+
+        INDICES_LIST=~{write_lines(select_all([vcf_index]))}
+        if [ -s $INDICES_LIST ]; then
+            # if index exists at this path, copy it to current working dir
+            if gsutil -q stat ~{automatic_vcf_index}; then
+                gsutil cp ~{automatic_vcf_index} .
+            fi
+        else
+            # vcf index was supplied, symlink it to current working dir
+            read INDEX_FILE < $INDICES_LIST
+            ln -s "$INDEX_FILE" .
+        fi
+
+        export GCS_OAUTH_TOKEN=`gcloud auth application-default print-access-token`
+        bcftools query -l ~{vcf} | wc -w > ~{num_samples_file}
+        # get num records from index. If index is wrong kind or missing, build index on the fly.
+        {
+            bcftools index --nrecords ~{vcf} || {
+                bcftools index -f -t -o "$(basename ~{vcf}).tbi" ~{vcf}
+                bcftools index --nrecords ~{vcf}
+            }
+        } > ~{num_records_file}
+    >>>
+
+    output {
+        Int num_records = read_int(num_records_file)
+        Int num_samples = read_int(num_samples_file)
+        Int num_entries = num_records * num_samples
```

Comment:
Had no idea, but that's good enough for me. 

To play devil's advocate, we'd want to look at the spec since someone could write another WDL interpreter with a different implementation. The spec for the `Int` type is:
```
$integer = [1-9][0-9]*|0[xX][0-9a-fA-F]+|0[0-7]*
```
I'm not sure I would take this literally. If I'm reading this right, these patterns technically permit unbounded non-negative integers…

Reply:
I went back and double-checked the source of this info. It's coded into the development and 1.1 WDL specs. 64-bit integer support was merged into cromwell in response to an issue when smaller ints had caused problems.